common/test/acceptance/tests/lms/test_lms_cohorted_courseware_search.py

```diff
@@ -110,6 +110,16 @@ def _goto_staff_page(self):
         self.assertEqual(staff_page.staff_view_mode, 'Staff')
         return staff_page
 
+    def _search_for_term(self, term):
+        """
+        Search for term in course and return results.
+        """
+        self.course_home_page.visit()
+        self.course_home_page.search_for_term(term)
+        course_search_results_page = CourseSearchResultsPage(self.browser, self.course_id).wait_for_page()
```

Comment:
@HarryRein could you address @robrap's suggestions for the Bok Choy tests, since I think you wrote this (or was it @dianakhuang?)